zhaquirks/ikea/__init__.py

```diff
@@ -87,6 +184,34 @@ class PowerConfiguration2AAACluster(DoublingPowerConfigurationCluster):
     }
 
 
+class PowerConfigurationND2AAACluster(PowerConfigurationCluster):
```

Comment:
Power configuration needs to be cleaned up.

I'll have a look at this later, but I think we could have "no doubling" classes like this as a base and then have the "doubling classes" just inherit from that **and** `DoublingPowerConfigurationCluster`.